### ISSUE TITLE
Problem with force=true due to missing .o .ji files

### ISSUE THREAD

**Issue by TsurHerman** [JuliaLang/PackageCompiler.jl#37]
The build process seems to be working right , but when  I set force = True
it tries to copy "sys.o" "sys.ji", "inference.o", "inference.ji" from PackageCompiler sysimg folder
to the system sys_img folder.  
All of these files are missing.
I located sys.o  in tmp_v0.6.2 folder but the rest are missing .

The only file that is present is sys.dylib , which If I copy manually to the system sysimg folder seems
to be working.

**Comment by SimonDanisch** [comment]
Ah, I might have destroyed it again, since "force" is so destructive to test...Let me check!

**Comment by TsurHerman** [comment]
Is it necessary? isn't sys.dylib sufficient, I am pretty sure that the object file is an intermediate un-nessaceray file . and likewise the ji file is baked into the dylib. 

I just deleted those files from the system sysimg folder for a fresh Julia install and ... nothing happened.

I think the only file that should be copied is sys.dylib.
+
when compiling `julia_flags` should include a flag to use the saved vanilla sysimg.
Also snoopcompile should use another process with the saved vanilla sysimg

**Comment by SimonDanisch** [comment]
Yeah I'm trying to confirm that ;) Have you used it that way and gotten all the speed ups?

**Comment by TsurHerman** [comment]
yes .. it feels even snappier, or maybe it is just my imagination ..

**Comment by SimonDanisch** [comment]
Lol... the difference should be pretty big if things work correctly... E.g. for GLVisualize it's multiple minutes vs ~1 second.... Depends on the file you're snooping though, if that has a bad coverage, you won't really get a big speed up.

**Comment by TsurHerman** [comment]
No I mean vanilla Julia with or without the potentially "irrelevant" files

**Comment by TsurHerman** [comment]
And yes I managed to use it for awhile to achieve really nice speedups especially for GLV ,as you mentioned it is a complete game changer.
But now I am fumbling with weird errors trying to simply go back to my previous successful attempt.

I made a "Workout" dummy module that creates a window loads a mesh parses an ini file.. it seems that PackageCompiler lived to its name and indeed the performance was as if I was running it for the second time.

I encounter weird LLVM.dylib segfaults at the end of the compilation phase.. still trying to pin point MWE

**Comment by SimonDanisch** [comment]
That probably means the code you're precompiling is not safe for statically compiling it!
You probably already have seen: https://github.com/JuliaLang/PackageCompiler.jl#troubleshooting

**Comment by TsurHerman** [comment]
And another point , it might make sense to put
--precompiled= no
 --compilecache=no

both in the snooping and in the compiling phase

**Comment by TsurHerman** [comment]
The problem was due to GLvisualize GLAbstraction GLWindow .. on a tagged release.
The problem goes away on latest master for those packages .. can you please tag versions for those packages that pass PackageCompiling
+ 
If you are looking for a minimum working example for failures then I think I just found one

**Comment by SimonDanisch** [comment]
```
And another point , it might make sense to put
--precompiled= no
--compilecache=no
```
Did you confirm that it still works with those? I can imagine, that they are actually needed, and there is no reference to using them in https://juliacomputing.com/blog/2016/02/09/static-julia.html
